phpunit.xml

```diff
@@ -19,7 +19,7 @@
         <server name="BCRYPT_ROUNDS" value="4"/>
         <server name="CACHE_DRIVER" value="array"/>
         <server name="DB_CONNECTION" value="mysql"/>
-        <server name="DB_DATABASE" value="mysql_testing"/>
+        <server name="DB_DATABASE" value="migration"/>
```

Comment:
Switch the `DB_DATABASE` back to `mysql_testing` for the GitHub action to work correctly.

```suggestion
        <server name="DB_DATABASE" value="mysql_testing"/>
```